languages/csharp/exercises/concept/nullability/.docs/instructions.md

```diff
@@ -0,0 +1,46 @@
+In this exercise you'll be working on a simple helper class that manipulates nullable values.
+
+### 1. Detect null values
+
+Write a static method called `IsNull` to detect if a string is `null` or not.
+
+``` csharp
+Nullable.IsNull("hello") // should return `false`
+Nullable.IsNull(null)    // should return `true`
+```
+
+### 2. Letting the runtime detect null values
+
+The C# runtime is able to detect when code calls a method from a `null` value. 
+It throws a `NullReferenceException`.
+
+Write a static method called `TotalLength` to sum the lengths of two strings passed as parameters.
+It should throw `NullReferenceException` when one of the strings is `null`.
```

Comment:
I understand why you added this test, but I think that in this case it does not make much sense to throw a `NullReferenceException`, as that is something you'll never do in any production code. Perhaps it would make sense not to have a separate task for the `NullReferenceException`, but for it to be sort of automatically included in a task similar to the next task, where one has to deal with `null` values. Students failing to do proper `null` checking will find a `NullReferenceException` being thrown. If we mention what happens when calling a member on a `null` value in the `introduction.md` document, I think that's fine for learning purposes, We can also add hints to the `hints.md` how to fix any `NullReferenceException`s.